### PR TITLE
OBPIH-3282 Refactor custom links in megamenu to use the same configur…

### PR DESCRIPTION
**Pull request by DominikaHatala**
…ation as the react menu

### CODE REVIEW

File: grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -1124,9 +1124,8 @@ openboxes {
             label = "customLinks.label"
             defaultLabel = "Custom Links"
             menuItems = [
-                [label: "requestItemCreation.label", defaultLabel: "Request Item Creation", href: "", target: "_blank"], // Fill in href
-                [label: "trainingVideos.label", defaultLabel: "Training Videos", href: "", target: "_blank"], // Fill in href
-                [label: "tableroNuevo.label", defaultLabel: "Tablero Nuevo", href: "/${appName}/tablero"],
+                    [label: "requestItemCreation.label", defaultLabel: "Request Item Creation", href: "https://goo.gl/forms/x1red4Q7fL0ThH253", target: "_blank"],
```
Comment (awalkowiak) [inline grails-app/conf/Config.groovy:1127]: @DominikaHatala I think this should not be here.

**Review by awalkowiak — CHANGES_REQUESTED** [review]

File: grails-app/conf/Config.groovy (hunk 1 above)
Comment (DominikaHatala) [reply grails-app/conf/Config.groovy:1127]: @awalkowiak why not?

Comment (awalkowiak) [reply grails-app/conf/Config.groovy:1127]: I meant the value of href, because we don't want to show such links in a public repo.

Comment (DominikaHatala) [reply grails-app/conf/Config.groovy]: @awalkowiak Yeah sure, I've just updated PR.

**Review by jmiranda — COMMENTED** [review]
Approved

**Pull request merged**